modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -61,6 +61,10 @@ This section provides the necessary details that enable you to control egress tr
 |443
 |Provides access to the `odo` CLI tool that helps developers build on OpenShift and Kubernetes.
 
+|`registry.connect.redhat.com`
+|443, 80
+|Required for all third-party images and certified operators.
```

Comment:
Init cap for "Operators", per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#operator).